staging/src/k8s.io/apiextensions-apiserver/examples/client-go/main.go

```diff
@@ -42,7 +41,7 @@ func main() {
 	flag.Parse()
 
 	// Create the client config. Use kubeconfig if given, otherwise assume in-cluster.
-	config, err := buildConfig(*kubeconfig)
+	config, err := clientcmd.BuildConfigFromFlags("", *kubeconfig)
```

Comment:
`BuildConfigFromFlags` writes `"Neither --kubeconfig nor --master was specified.  Using the inClusterConfig.  This might not work."`. We don't have `--master` here. We might want to add that flag here as well.

Reply:
@sttts Good idea, added the `--master` flag. Can you have another look, thanks!